views/js/runner/helpers/stats.js

```diff
@@ -47,6 +47,8 @@ define([
 
             } else if (isItemCurrentlyAnswered && !context.itemAnswered) {
                 stats.answered++;
+            } else if(!isItemCurrentlyAnswered && !context.itemAnswered) {
+                stats.answered++;
```

Comment:
I don't understand the purpose here. The result of this code will be, no matter the value of `isItemCurrentlyAnswered`:

```javascript
if (!context.itemAnswered) {
    stats.answered++;
}
```

I don't think this is a wanted behavior, as the item will always be considered answered. This will introduce regression.